seed/challenges/00-getting-started/getting-started.json

```diff
@@ -34,7 +34,7 @@
         ],
         [
           "//i.imgur.com/vLNso6h.jpg",
-          "An illustration showing that you will learn HTML5, CSS3, JavaScript, Databases, Git, Node.js, React and D3.",
+          "An illustration showing that you will learn HTML5, CSS3, JavaScript, Databases, Git, Node.js, React, and D3.",
```

Comment:
The next sentence does not have an Oxford comma either. Can you remove the added Oxford comma here for consistency?